operations/node_management/Wipe_NCN_Disks.md

```diff
@@ -91,23 +101,19 @@ deleting the Ceph volumes, and then wiping the disks and RAIDs.
 
     Examine the output. There should be no running `ceph-osd` processes or containers.
 
-1. Remove the Ceph volume groups.
-
-    ```bash
-    vgremove -f -v --select 'vg_name=~ceph*'
-    ```
-
-1. Perform the [Basic Wipe](#basic-wipe) procedure.
+1. Perform the [basic wipe](#basic-wipe) procedure.
 
 ## Full-Wipe
 
-This section is the preferred method for all nodes. A full wipe includes deleting the Ceph volumes (where applicable), stopping the
-RAIDs, zeroing the disks, and then wiping the disks and RAIDs.
+This section walks a user through cleanly stopping all running services that require partitions, as well as completely scrubbing and removing
+the node.
```

Comment:
Sorry, I should clarify. My concern isn't the word system vs node. It just seems like this procedure doesn't do either. It scrubs the node, sure. I guess it removes the node from the Ceph cluster or the Kubernetes cluster. If that's the removal intended here, I'd state that instead "removing the node from the Ceph or Kubernetes cluster (as appropriate for the node type)."